packages/react/src/components/CodeSnippet/CodeSnippet.js

```diff
@@ -187,12 +187,12 @@ function CodeSnippet({
         {...rest}
         align={align}
         onClick={handleCopyClick}
-        aria-label={deprecatedAriaLabel || ariaLabel}
+        aria-label={ariaLabel}
```

Comment:
Did you mean to remove this? I think we may still want it in case anyone was using the `ariaLabel` prop?